src/av-vertrag.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: layout
+title: Auftragsverarbeitung
```

Comment:
und diese wirft:

 1 of 1 unhandled error
Unhandled Runtime Error

TypeError: IntersectionObserver.observe: Argument 1 is not an object.
Source

http://localhost:3000/static/js/page-prerenderer.js [:25:4]

@http://localhost:3000/static/js/page-prerenderer.js:25:4